.gitignore

```diff
@@ -11,6 +11,7 @@ src/goto-analyzer/taint_driver_scripts/.idea/*
 /*.creator.user
 /*.files
 /*.includes
+/*.idea
```

Comment:
My apologies for the nit-pick, but -- in absence of an additional section -- could some lexicographic ordering be used? That's just moving it up one line...

Reply:
Done